packages/client/webapp/src/App.vue

```diff
@@ -23,5 +27,22 @@ const isAuthenticated = computed(() => auth.isAuthenticated);
 
 <template>
 	<Header />
-	<router-view v-if="isAuthenticated"></router-view>
+	<div :class="sidebarPosition" v-if="isAuthenticated">
+		<Sidebar
+			:sidebar-position="sidebarPosition"
+			@updateSidebarPosition="(newPosition) => (sidebarPosition = newPosition)"
+		/>
+		<router-view />
+	</div>
 </template>
+
+<style scoped>
+div {
```

Comment:
Sorry, you can do that, you need to name that div something more specific, as it will affect all the other `div` from within the `<router-view>`

Reply:
I don't think that's true, scoped files don't affect descendent elements.

For example, the `div` surrounding the buttons on the home page:

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/16928557/194626354-afa390d0-c648-4211-9b19-2a1e5a3028c4.png">

It's inside the `router-view`, but it doesn't inherit any of these styles from App.vue:

<img width="678" alt="image" src="https://user-images.githubusercontent.com/16928557/194626490-db5cb4db-7e8c-4dd0-8e14-c338c271d63a.png">

Reply:
I agree, I was working on an old version of this branch